app/controllers/events/view/sessions/list.js

```diff
@@ -161,6 +168,44 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async updateRating(session, rating) {
+      if (session.feedbacks.length) {
+        let feedback = session.feedbacks.get('firstObject');
+        feedback.set('rating', rating);
+        this.set('isLoading', true);
+        await feedback.save()
+          .then(() => {
```

Comment:
This is an incorrect usage of `async-await`.
Please read-up more on it https://javascript.info/async-await